platform_versioned_docs/version-24.2/enterprise/studios.mdx

```diff
@@ -5,7 +5,7 @@ date: "12 Jun 2024"
 tags: [docker, compose, kubernetes, data, studios, deployment]
 ---
 
-Enable Studios as part of your Seqera Platform instance. You must have Data Explorer enabled to use Data Studios. Only the Amazon AWS public cloud is currently supported.
+Enable Studios as part of your Seqera Platform instance. You must have Data Explorer enabled to use Studios. Only the Amazon AWS public cloud is currently supported.
 
 :::caution 
 You must complete the upgrade of your Platform Enterprise installation to version 24.1.4 before enabling and configuring Data Studios.
```

Comment:
```suggestion
You must complete the upgrade of your Platform Enterprise installation to version 24.2 before enabling and configuring Studios.
```

Let's start requiring to upgrade to v24.2 so we start collecting telemetry data.